# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qt63d/hotfix-cstdint.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

/usr/src/t2-src/src.qt63d.default.20240129.201701.19180.portable2/qt3d-everywhere-src-6.2.7/src/3rdparty/assimp/src/code/AssetLib/FBX/FBXBinaryTokenizer.cpp:475:61: error: 'uint32_t' is not a member of 'std'; did you mean 'wint_t'?

--- qt3d-everywhere-src-6.2.7/src/3rdparty/assimp/src/code/AssetLib/FBX/FBXBinaryTokenizer.cpp.vanilla	2024-01-29 20:19:30.841006206 +0100
+++ qt3d-everywhere-src-6.2.7/src/3rdparty/assimp/src/code/AssetLib/FBX/FBXBinaryTokenizer.cpp	2024-01-29 20:20:01.379003658 +0100
@@ -50,7 +50,7 @@
 #include "FBXTokenizer.h"
 #include "FBXUtil.h"
 #include <assimp/defs.h>
-#include <stdint.h>
+#include <cstdint>
 #include <assimp/Exceptional.h>
 #include <assimp/ByteSwapper.h>
 #include <assimp/DefaultLogger.hpp>
